src/lib/list/selection-list.ts

```diff
@@ -170,11 +157,11 @@ export class MdListOption extends _MdListOptionMixinBase
 
   _handleFocus() {
     this._hasFocus = true;
-    this._renderer.addClass(this._element.nativeElement, FOCUSED_STYLE);
+    this.selectionList._setFocusedOption(this);
   }
 
   _handleBlur() {
```

Comment:
Since this is just toggling a boolean, it can also be done inline.

Reply:
Sure